gyp/pylib/gyp/common.py

```diff
@@ -454,6 +454,8 @@ def GetFlavor(params):
         return "aix"
     if sys.platform.startswith(("os390", "zos")):
         return "zos"
+    if sys.platform == "os400":
```

Comment:
I would feel more comfortable with the following to allow for future expansion like all other oses above.
```suggestion
    if sys.platform.startswith("os400"):
```

Reply:
Python has been moving away from versioning sys.platform. Linux changed to just "linux" in Python 3.3 and AIX changed to just "aix" in 3.8. Our changes (not yet upstream) set it to just os400 to follow this trend and since there was never any previous version that returned a version-prefixed "os400", there _should_ be no need to use startswith. However, I can make these changes if you still prefer.

https://docs.python.org/3/library/sys.html#sys.platform

Reply:
Cool.